cnf-certification-test/accesscontrol/suite.go

```diff
@@ -438,7 +439,7 @@ func testPodServiceAccount(env *provider.TestEnvironment) {
 	failedPods := []string{}
 	for _, put := range env.Pods {
 		ginkgo.By(fmt.Sprintf("Testing service account for pod %s (ns: %s)", put.Name, put.Namespace))
-		if put.Spec.ServiceAccountName == "" {
+		if put.Spec.ServiceAccountName == defaultServiceAccount {
```

Comment:
update catalog test description with detailed behavior (e.g. that we should not use the default service account)

Reply:
Updated. Please review again!